### PR TITLE
test: add e2e test for geo

### PR DESCRIPTION
**Pull request by AaronZyLee**
#### Description of changes

Add e2e test cases for amplify geo category(map and place index). The commands are `add`, `update` and `remove`.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-e2e-tests/src/__tests__/geo_1.test.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import {
```
Comment (phani-srikar) [inline packages/amplify-e2e-tests/src/__tests__/geo_1.test.ts:1]: can we name the test files as `geo-add.test.ts` or equivalent to make it more clear?

**Review by phani-srikar — CHANGES_REQUESTED** [review]
Nice set of e2e tests to cover the default walkthroughs. 
I have listed out some cases mostly related to changing the `isDefault` setting of maps/place-indices, that would be good to have. The default setting is important to cover in the sense that the Amplify Frontend Clients will use it to make all default SDK calls.

File: packages/amplify-e2e-tests/src/__tests__/geo_1.test.ts (hunk 1 above)
Comment (AaronZyLee) [reply packages/amplify-e2e-tests/src/__tests__/geo_1.test.ts:1]: Sure. The previous naming was just following the CLI's convention.

**Review by phani-srikar — APPROVED** [review]
LGTM! Found nothing of major concern.

**Pull request merged**